src/HIE/Bios/Cradle.hs

```diff
@@ -796,6 +811,17 @@ cabalGhcDirs l cabalProject workDir = do
   where
     projectFileArgs = projectFileProcessArgs cabalProject
 
+cabalPathCompilerPath :: LogAction IO (WithSeverity Log) -> FilePath -> CradleLoadResultT IO (Maybe FilePath)
+cabalPathCompilerPath l workDir = do
+  liftIO (getCabalVersion l workDir) >>= \case
+    Just cabal_version | cabal_version >= makeVersion [3,14] -> do
+      compiler_info <- readProcessWithCwd_ l workDir "cabal" ["path", "--compiler-info"] ""
+      let compiler_path = asum $ stripPrefix "compiler-path: " <$> lines compiler_info
```

Comment:
Why not use `--output-format=json` and then access the keys properly? I'm not sure `key-value` is very suitable for machine reading?

Reply:
Thanks for pointing this out, I also think we should the json output format.

Reply:
Sure. Do I use `aeson` or something else?
I'm only familiar with that one but would prefer a *fast* parser. We're already slow as is